planet/api/orders.py

```diff
@@ -247,15 +250,17 @@ async def aggregated_order_stats(self) -> dict:
     async def download_asset(
         self,
         location: str,
+        order_id: str,
```

Comment:
I would prefer the user not need to know the order id if they already know the location of the asset they want to download. That just makes more work for them.

Reply:
I'm not sure I fully understand how users interact with this module on the Python side yet, but from the looks of it, I thought users only queried `self.download_order()` to download their order, for which they need to provide an Order ID. I suppose the Order ID could get stored in the class object, `self`, and get updated every time they request to download a new order. I don't think my current changes affect the user on the CLI side of things.

Thoughts?